src/react-components/room/LeaveRoomModal.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styles from "./LeaveRoomModal.scss";
+import { useIntl, defineMessages } from "react-intl";
+import PropTypes from "prop-types";
+import { Modal } from "../modal/Modal";
+import { CloseButton } from "../input/CloseButton";
+import { Button } from "../input/Button";
+
+const messages = defineMessages({
+  "join-room-message": {
+    id: "leave-room-dialog.join-room.message",
+    defaultMessage: "Joining a new room will leave this one. Are you sure?"
+  },
+  "join-room-confirm": {
+    id: "leave-room-dialog.join-room.confirm",
+    defaultMessage: "Join Room"
+  },
+  "create-room-message": {
+    id: "leave-room-dialog.create-room.message",
+    defaultMessage: "Creating a new room will leave this one. Are you sure?"
+  },
+  "create-room-confirm": {
+    id: "leave-room-dialog.create-room.confirm",
+    defaultMessage: "Leave Room"
```

Comment:
The title and confirmation button text for this case are awkward. I know this is taken from what we already have, but it seems like the button text or title should be something like "Create Room" or possibly "Leave and Create New Room"